src/tasks/withdraw/value.ts

```diff
@@ -2,8 +2,14 @@ import axios from "axios";
 import { BigNumber, constants } from "ethers";
 
 import { OrderKind } from "../../ts";
+import { SupportedNetwork } from "../ts/deployment";
 import { TokenDetails } from "../ts/erc20";
 
+// canonical-weth is not typed
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const wrappedNativeTokenByNetworkId = require("canonical-weth")
```

Comment:
Shouldn't `import WethNetworks from "canonical-weth/networks.json"` work here? It should even be typed `as const` so `WethNetworks[923846293]` would produce a compile error.

Reply:
Yep, much better now!